README.md

```diff
@@ -244,6 +246,40 @@ Create a CGFloat from a Float or an Integer:
 let imageViewTop = 15.f
 ```
 
+### CGPoint extension
+
+Add two `CGPoint`:
+
+```swift
+var point1 = CGPoint(x: 10, y: 10)
+let point2 = CGPoint(x: 10, y: 10)
+print(point1 + point2) // CGPoint(x: 20, y: 20)
+
+point1 += point2
+print(point1) // CGPoint(x: 20, y: 20)
+```
+
+Substract two `CGPoint`:
+
+```swift
+var point1 = CGPoint(x: 10, y: 10)
+let point2 = CGPoint(x: 10, y: 10)
+print(point1 - point2) // CGPoint(x: 0, y: 0)
+
+point1 -= point2
+print(point1) // CGPoint(x: 0, y: 0)
+```
+
+Multiply a `CGPoint` with a scalar:
+
+```swift
+var point1 = CGPoint(x: 10, y: 10)
+print(point1 * 2) // CGPoint(x: 20, y: 20)
+
+point1 += 2
```

Comment:
Why didn't you use the `*=` here (and in the `CGSize` extension as well) for multiplying by scalar? Typo?